### PR TITLE
Fixed shuffle button opacity UI bug

### PR DESCRIPTION
**Pull request by 0x416c6578**
Parameterised shuffle state into initPlayback for potentially passing the shuffle state into the player in the future

#### What is it?
- [x] Bugfix (user facing)
- [ ] Feature (user facing)
- [ ] Codebase improvement (dev facing)
- [ ] Meta improvement to the project (dev facing)

#### Description of the changes in your PR

- Fixes the shuffle mode button UI bug and parameterizes the shuffle mode in `initPlayback` (to potentially allow passing the saved shuffle state into the player in the future)

#### Fixes the following issue(s)

- Fixes https://github.com/TeamNewPipe/NewPipe/issues/6825

#### APK testing 

The APK can be found by going to the "Checks" tab below the title. On the left pane, click on "CI", scroll down to "artifacts" and click "app" to download the zip file which contains the debug APK of this PR.

#### Due diligence
- [x] I read the [contribution guidelines](https://github.com/TeamNewPipe/NewPipe/blob/HEAD/.github/CONTRIBUTING.md).

### CODE REVIEW

File: app/src/main/java/org/schabi/newpipe/player/Player.java [hunk 1]
```diff
@@ -748,10 +749,12 @@ private void initPlayback(@NonNull final PlayQueue queue,
                               final float playbackPitch,
                               final boolean playbackSkipSilence,
                               final boolean playOnReady,
-                              final boolean isMuted) {
+                              final boolean isMuted,
+                              final boolean shuffleEnabled) {
         destroyPlayer();
         initPlayer(playOnReady);
         setRepeatMode(repeatMode);
+        onShuffleModeEnabledChanged(shuffleEnabled);
```
Comment (Stypox) [inline app/src/main/java/org/schabi/newpipe/player/Player.java:757]: Just do `onShuffleModeEnabledChanged(false)` here and remove all of the other code changes, that would eventually be done in another PR saving the shuffle state

**Review by Stypox — CHANGES_REQUESTED** [review]
Are you sure this doesn't reset the queue shuffle mode when switching players?

File: app/src/main/java/org/schabi/newpipe/player/Player.java (hunk 1 above)
Comment (0x416c6578) [reply app/src/main/java/org/schabi/newpipe/player/Player.java:757]: yeah I did it in such a way that the intent could be changed to add the shuffle mode but if that isn't needed I'll switch to what you recommended

**Comment by Stypox** [comment]
@0x416c6578 have you done anything new? Should this be reviewed? Sorry for not noticing earlier, I didn't see the notification

**Comment by 0x416c6578** [comment]
I just realised I assumed it had been fixed by someone else since it wasn't a problem in the app on my phone, but then I remembered I am using builds from my own fork with the above fix lol. The only change I ended up making was adding the line `onShuffleModeEnabledChanged(false);` to handleIntent.

**Comment by Stypox** [comment]
Thank you, that's good. Could you verify this in the multiple conditions that `initPlayback` is called?

> Are you sure this doesn't reset the queue shuffle mode when switching players?

Then this can be merged

**Comment by litetex** [comment]
Reminder ping @0x416c6578

**Comment by 0x416c6578** [comment]
for reference the changes in this PR are working, so this could be merged. i'll just open another one with the aforementioned changes when i have some time :/

**Review by litetex — DISMISSED** [review]
Rewrote the fix as mentioned and tested it with a few scenarios.
Approved from my side.
If anyone else could test this PR that would be great.

**Comment by tsiflimagas** [comment]
I'm not sure if by "anyone else" you meant some other dev or any user, but I tested this and seems to be working fine.

**Review by litetex — APPROVED** [review]
